### PR TITLE
gui: add time app used & refactor.

### PR DESCRIPTION
**Pull request by Zangetsu38**
# About:
Add time app used inside app context menu in information.
Refactor code for can using new value for no get double value exist for same things.
Using so now app list by user.

- move cfg to new value.

# Result
- is fake time used here off course, is just for test etc :D 
image

### CODE REVIEW

File: vita3k/kernel/include/kernel/object_store.h [hunk 1]
```diff
@@ -49,7 +49,7 @@ class ObjectStore {
     }
 
     template <typename T, typename... Args>
-    bool create(Args &&... args) {
+    bool create(Args &&...args) {
```
Comment (pent0) [inline vita3k/kernel/include/kernel/object_store.h:52]: hm

Comment (pent0) [reply vita3k/kernel/include/kernel/object_store.h:52]: Unrelated format changes not to be here.

Comment (Zangetsu38) [reply vita3k/kernel/include/kernel/object_store.h:52]: is done by format bat ^^

Comment (pent0) [reply vita3k/kernel/include/kernel/object_store.h:52]: bbb vv zz

**Review by pent0 — APPROVED** [review]

**Pull request merged**